### DISCUSSION TITLE
Is there any place where I can watch the speechs from the conf?

### DISCUSSION THREAD

**Discussion by wilsonneto-swe** [vercel/next.js#18494] [Help]
Hi guys,
 Is there any place where I can watch the speechs from the conf?

Thanks so much,
best regards.

**Comment by eric-hc** [comment]
[Here is the keynote](https://www.youtube.com/watch?v=1-NzQ9ObsfM&ab_channel=Vercel), but I bet they will likely all be available on their YouTube channel at some point.

**Comment by ammuhaysin** [comment]
From the Next.js discord:

![nextconf](https://user-images.githubusercontent.com/51197058/97627741-d9d66b00-1a3c-11eb-9d6e-5d10ae7e4c12.JPG)
